### ISSUE TITLE
[Picker][NVDA] Position and size of search suggestions not announced correctly

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#5846]
NVDA fails to announce the proper size and position of the search suggestion results displayed while entering tags in picker component.
Windows narrator works fine. Issue is with NVDA+Chrome only.

Repro can be seen here:
https://developer.microsoft.com/en-us/fabric#/components/pickers 

![image](https://user-images.githubusercontent.com/11937034/43706127-29529e80-9982-11e8-9b20-2e8da2234feb.png)

NVDA+Chrome response:

![image](https://user-images.githubusercontent.com/11937034/43818154-f6e34416-9afa-11e8-9d8c-60e0d2ef7222.png)

NVDA+Firefox response:

![image](https://user-images.githubusercontent.com/11937034/43817938-cf40f986-9af9-11e8-9f3d-1a12f2da268a.png)

**Comment by natalieethell** [comment]
Hey @prmittal , thanks for submitting this issue! As mentioned in [this issue](https://github.com/OfficeDev/office-ui-fabric-react/issues/5716), our primary support scenario is Narrator on Edge. We're happy to make fixes in cases where we are not following Aria spec, but if we are otherwise adhering to W3C Aria spec, we don't make fixes for specific screen readers (a bug should be filed with the screen reader instead). 

Given that it is working properly for Edge + Narrator, and we have the correct attributes `role=list` on the suggestions container and `role=listitem` on each of the list items in the picker, this seems like a bug specific to NVDA.

If you'd really like this fixed, I would say consider opening an issue with NVDA here: https://github.com/nvaccess/nvda/issues. 

I'll go ahead and close this, but let us know if you have any further questions. Thanks!
